Innovative Flix/force-app/main/default/objects/Episode__c/validationRules/Number_Of_Season_Greather_Than_0.validationRule-meta.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Create other validation than on the Season object

Reply:
Done, on Description field